json-api.md

```diff
@@ -0,0 +1,105 @@
+# JSON API
+
+## Get Access Points
+
+Path: `/api/v1/access_points`
+
+Method: `GET`
+
+Response: Array AccessPoint
+
+Response Code: `200`
+
+### Examples
+
+#### Response
+
+```json
+
+[
+  {
+    ssid: "Free WiFi!",
+    frequency: "blah",
+    flags: [],
+    signal_percent: 100
+  },
+  {
+    ssid: "Imperial Star Destroyer",
+    frequency: "blah",
+    flags: [],
+    signal_percent: 100
+  }
+]
+```
+
+## Connect to WiFi
+
+Path: `/api/v1/access_points/connect`
+
+Method: `POST`
+
+Request: Array WiFiConfiguration
+
+Response: Empty
+
+Response Code: `202`
+
+### Examples
+
+#### Request
+
+```json
+[
+  {
+    "ssid": "Millennium Falcon",
+    "password": "Chewbacca",
+    "frequency": "2.4 GHz channel 6",
```

Comment:
I think it's safe to remove `frequency` since `vintage_net` can't do anything with it anyway. SSID, key_mgmt, password are good. I also think that for now we just do `none` and `wpa_psk` key management. I'm sure that @ConnorRigby may add WPA EAP at some point.